modules/apps/frontend-js/frontend-js-test/src/testFunctional/A11yTool.testcase

```diff
@@ -0,0 +1,51 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "User Interface";
+	property testray.main.component.name = "User Interface";
+
+	setUp {
+		property osgi.modules.includes = "frontend-js-clay-sample-web";
+
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Classic Theme Page");
```

Comment:
Make sure copied artifacts are adjusted to the testcase. This is not a theme testcase.